docs/reference/content/bson/pojos.md

```diff
@@ -0,0 +1,262 @@
++++
+date = "2017-04-10T14:27:51-04:00"
+title = "POJOs"
+[menu.main]
+  parent = "BSON"
+  weight = 45
+  pre = "<i class='fa fa-star'></i>"
++++
+
+## POJOs - Plain Old Java Objects
+
+The 3.5 release of the driver adds POJO support via the [`PojoCodec`]({{<apiref "org/bson/codecs/pojo/PojoCodec.html">}}), which allows for 
+direct serialization of POJOs to and from BSON. Internally, each `PojoCodec` utilizes a
+[`ClassModel`]({{<apiref "org/bson/codecs/pojo/ClassModel.html">}}) instance to store metadata about how the POJO should be serialized.
+
+A `ClassModel` for a POJO includes:
+
+  * The class of the POJO.
+  * A new instance factory. Handling the creation of new instances of the POJO. By default it requires the POJO to have an empty constructor.
+  * Field information, a list of [`FieldModel`]({{<apiref "org/bson/codecs/pojo/FieldModel.html">}}) instances that contain all the field metadata.
```

Comment:
The doc should state what type of fields are serialized by default: all non-transient, non-final fields, I believe